docs/mp/persistence.adoc

```diff
@@ -0,0 +1,1430 @@
+  elements of your project's `pom.xml` file's `<dependencies>` element:
++
+[source,xml]
+----
+<dependency>
+    <groupId>jakarta.persistence</groupId>
+    <artifactId>jakarta.persistence-api</artifactId>
+    <scope>provided</scope> <!--1-->
+</dependency>
+<dependency>
+    <groupId>io.helidon.integrations.cdi</groupId>
+    <artifactId>helidon-integrations-cdi-jpa</artifactId>
+    <scope>runtime</scope> <!--2-->
+</dependency>
+----
+<1> The `scope` is `provided`, which ensures that the
+    https://jakarta.ee/specifications/persistence/3.0/apidocs/jakarta.persistence/jakarta/persistence/package-summary.html[JPA
+    classes required for compilation] are available at compile time.
+<2> The `scope` is `runtime`, which ensures that Helidon's core,
+    provider-independent Jakarta Persistence integration is available at
+    runtime.
+
+These `<dependency>` elements do not set up a JPA provider.  See
+details below for the JPA provider you have chosen to use.
+
+===== Setting Up Static Metamodel Generation [[JPA-MetaModel]]
+
+To generate and compile the Jakarta Persistence static metamodel for
+your application, regardless of whether you are using Hibernate ORM or
+Eclipselink, make sure the `<plugin>` element in the following code
+snippet is present as a child element of the
+`<pluginManagement><plugins>` element sequence as shown below:
+
+[source,xml]
+----
+<pluginManagement>
+    <plugins>
+
+        <!-- ... -->
+
+        <plugin>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <executions>
+                <execution>
+                    <id>default-compile</id>
+                    <configuration>
+                        <annotationProcessorPaths>
+                            <annotationProcessorPath>
+                                <groupId>org.hibernate.orm</groupId>
+                                <artifactId>hibernate-jpamodelgen</artifactId> <!--1-->
+                                <!--
+                                    Always check
+                                    https://search.maven.org/artifact/org.hibernate.orm/hibernate-jpamodelgen
+                                    for up to date versions and make
+                                    sure the version used is that of
+                                    Hibernate ORM itself.
+                                -->
+                                <version>6.1.1.Final</version>
+                            </annotationProcessorPath>
+                        </annotationProcessorPaths>
+                    </configuration>
+                </execution>
+            </executions>
+        </plugin>
+
+        <!-- ... -->
+
+    </plugins>
+</pluginManagement>
+----
+<1> This adds the `hibernate-jpamodelgen` jar, which contains a
+    https://github.com/hibernate/hibernate-orm/blob/6.1.1/tooling/metamodel-generator/src/main/java/org/hibernate/jpamodelgen/JPAMetaModelEntityProcessor.java#L59[Java
```

Comment:
There are various versioned links to hibernates in this document, should the hibernate version and common attributes be defined in includes/attributes.adoc ? Or maybe within this document ?

Reply:
Whichever you like